.github/workflows/test-e2e.yml

```
@@ -12,6 +12,8 @@ on:
 jobs:
   test-e2e-docker:
```

[uber nit] it would be nice if this matched the make command. 

```suggestion
  test-docker-e2e:
```

or we modify the make command to match the workflow name